reana_job_controller/k8s.py

```diff
@@ -181,21 +193,42 @@ def watch_jobs(job_db, config):
                 )
                 job_db[job.name]['pod'].delete()
                 job_db[job.name]['deleted'] = True
-
+                channel.basic_publish(exchange='',
```

Comment:
The 3 calls to basic_publish() should be sent to another queue (currently there's only 'yadage-jobs'), and then later consumed for keeping logs.